BCStudentSoftwareDevTeam/celts
# Minor page 989

# Changes
- Created a new html page at `/profile/neillz/cceMinor`
- Created new logic and controller functions in the respective `minor.py` and `routes.py` files.
- Added the functionality to view engagement history for courses and programs.

# Testing
- `database/reset_database.sh test`
- `tests/run_tests.sh`
- Visit the user profile page for any user
- Click on actions in the top right, and select "Manage CCE Minor"
- Test the express interest button, verify that selecting it updates your status to interest expressed.
- Click on the accordion and verify you see a selection of entries, for instance Summer 2021 should show Databases
- Click on an entry in the accordion and for courses, verify you see the course information, and for programs, see the volunteer's participation history.
